### DISCUSSION TITLE
Add an Item to Action Menu

### DISCUSSION THREAD

**Discussion by ghost** [Security-Onion-Solutions/securityonion#7925] [Unsupported Versions] [answered]
I would like to add an item to the Action menu. Where are the files that would allow me to do this? 
![SO_Action_Menu](https://user-images.githubusercontent.com/104913237/167656221-03b95ceb-d41d-44c8-92db-d0b226ee1fa2.jpg)

**Comment by greatapoc** [comment, marked as answer]
https://docs.securityonion.net/en/latest/soc-customization.html#soc-customization

**Reply by ghost** [reply]
Thank you very much!
